cell/src/main/java/jetbrains/jetpad/cell/message/MessageStyler.java

```diff
@@ -21,6 +21,8 @@
 import jetbrains.jetpad.values.Color;
 
 public class MessageStyler {
+  public static final Color YELLOWISH_BROWN = new Color(248, 231, 172);
```

Comment:
Shouldn't be WARNING_BACKGROUND? Especially if it's public. We could change it to a different color but it will still be warning background color.